src/components/users/UserList.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+import { useNavigate } from 'react-router-dom'
+import { getUsers, getUsersBySearchTerm } from "../../managers/users"
+import { User } from "./User"
+import "./Users.css"
+// import { UserSearch } from "./UserSearch"
+
+export const UserList = () => {
+
+  const [users, setUsers] = useState([])
+//   const [searchTerm, setSearchTerm] = useState('')
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    getUsers().then(usersData => setUsers(usersData))
+  }, [])
+//   useEffect(() => {
```

Comment:
No commented out code should go into the main branch

Reply:
removed